Exercises/4-count-types.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = arr => {
+  const types = {};
+  for (const el of arr) {
+    const t = el === null ? 'null' : typeof el;
+    if (!(t in types)) {
+      types[t] = 0;
+    }
+    types[t] += 1;
+  }
+  return types;
```

Comment:
See implementations here https://github.com/HowProgrammingWorks/DataTypes/pull/21#discussion_r324387734

Reply:
Thank you for reference to more elegance code